src/api/dashboard.php

```diff
@@ -1,2 +1,45 @@
 <?php
-header("HTTP/1.1 501 Not Implemented");
\ No newline at end of file
+header("HTTP/1.1 501 Not Implemented");
+function pdo_connect_mysql() {
+    // Update the details below with your MySQL details
+    $DATABASE_HOST = 'localhost';
+    $DATABASE_USER = 'root';
+    $DATABASE_PASS = '';
+    $DATABASE_NAME = 'phpticket';
+    try {
+    	return new PDO('mysql:host=' . $DATABASE_HOST . ';dbname=' . $DATABASE_NAME . ';charset=utf8', $DATABASE_USER, $DATABASE_PASS);
+    } catch (PDOException $exception) {
+    	// If there is an error with the connection, stop the script and display the error.
+    	exit('Failed to connect to database!');
+    }
+}
+
+switch ($_GET["action"]) {
+    case "watched-history":
+       //TBD
+        break;
+	case "recent-movie":
+		//query ticket join scheduled_movie and movie with ticket_status = "watched"
+		$stmt = $pdo->prepare('SELECT * FROM tickets WHERE id = ?');
```

Comment:
Since the request need a response with maximum number of 9 objects consider limit the numbers of result with "LIMIT" keyword